flow/Makefile

```diff
@@ -363,7 +363,7 @@ $(OBJECTS_DIR)/lib/merged.lib:
 # ==============================================================================
 $(OBJECTS_DIR)/klayout_tech.lef: $(TECH_LEF)
 	@mkdir -p $(OBJECTS_DIR)
-	sed '/OR_DEFAULT/d' $< > $@
+	cp $< $@
```

Comment:
Is there any need for this rule if it is a pure cp now?  Why not just use TECH_LEF

Reply:
The file is subsequently modified in follown rules.

We may dispel of those followon modifications at some point, but for now they are there